src/helpers/hotkeysManager.js

```diff
@@ -460,7 +461,17 @@ WebViewer(...)
    * @ignore
    */
   createToolHotkeyHandler(handler) {
+    const { getState } = this.store;
+
     return (...args) => {
+      const openElements = selectors.getOpenElements(getState());
+      const currentToolName = core.getToolMode().name;
+    
+      if (currentToolName=== window.Tools.ToolNames.SIGNATURE && openElements['signatureModal']) {
```

Comment:
Sorry just nit picking here, can we have a space before `===`

Reply:
Thanks for pointing this out, this has been fixed